ext/travis/test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "$TRAVIS_OS_NAME" = linux ]; then
+    uid="$(id -u)"
+    gid="$(id -g)"
+    tinyfs="$(pwd)/tinyfs"
+
+    sudo -i dd if=/dev/zero of=tinyfs.img bs=10MB count=1
+    sudo -i chmod 600 tinyfs.img
+    sudo -i mke2fs -F tinyfs.img
+    sudo -i mkdir "$tinyfs"
+    sudo -i mount -oloop tinyfs.img "$tinyfs"
+    sudo -i chown "$uid:$gid" "$tinyfs"
```

Comment:
@rbrw @ajroetker So the only command here that should need root permission is the `mount` command, but aside from that (and sidestepping your question about root permissions), this script as written will very likely cause problems on our unit test slaves since it does not appear to ever unmount the mounted filesystem. It's fine in Travis because Travis makes good use of containers where it's okay to affect global state on the test running machine.

I would prefer not to risk leaving un-unmounted filesystems around on our static Jenkins infrastructure (in the case of test failures, aborted jobs, etc). For what it's worth, in CI.Next we will have access to docker containers for use cases like this so if Travis can hold you over until we start to see some of the benefits from CI.Next let's stick with it for now.